modules/apps/29-fee/keeper/grpc_query.go

```diff
@@ -1,5 +1,65 @@
 package keeper
 
-// TODO
+import (
+	"context"
 
-//var _ types.QueryServer = Keeper{}
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/types/query"
+
+	"github.com/cosmos/ibc-go/modules/apps/29-fee/types"
+)
+
+var _ types.QueryServer = Keeper{}
+
+// IncentivizedPackets implements the IncentivizedPackets gRPC method
+func (k Keeper) IncentivizedPackets(c context.Context, req *types.QueryIncentivizedPacketsRequest) (*types.QueryIncentivizedPacketsResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
+	}
+
+	ctx := sdk.UnwrapSDKContext(c).WithBlockHeight(int64(req.QueryHeight))
+
+	packets := []*types.IdentifiedPacketFee{}
```

Comment:
This is a super subtle difference, but follows along with other code in the repo and is probably considered more "standard". Essentially its declaration vs initialisation.

```suggestion
	var packets []*types.IdentifiedPacketFee
```